File: chatwoot/python/src/UpdateAMessageExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
)

with ApiClient(configuration) as api_client:
    public_message_update_payload = models.PublicMessageUpdatePayload(
    )

    try:
        response = api.MessagesAPIApi(api_client).update_a_message(
            inbox_identifier="inbox_identifier_string",
            contact_identifier="contact_identifier_string",
            conversation_id=0,
            message_id=0,
            data=public_message_update_payload,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling MessagesAPIApi#update_a_message: %s\n" % e)

File: chatwoot/ruby/src/UpdateAMessageExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
end

public_message_update_payload = ChatwootClient::PublicMessageUpdatePayload.new

begin
    response = ChatwootClient::MessagesAPIApi.new.update_a_message(
        "inbox_identifier_string", # inbox_identifier
        "contact_identifier_string", # contact_identifier
        0, # conversation_id
        0, # message_id
        public_message_update_payload, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling MessagesAPIApi#update_a_message: #{e}"
end
